TaskLayer/SearchTask/PostSearchAnalysisTask.cs

```diff
@@ -580,6 +585,35 @@ private void WritePsmResults()
             }
         }
 
+        //for those spectra matching the same peptide/protein with same charge, save the one with highest score
+        private void SpectralLibraryGeneration()
+        {
+            var FilteredPsmList = Parameters.AllPsms
+               .Where(p => p.FdrInfo.PEP_QValue <= 0.01 && p.FdrInfo.QValueNotch <= CommonParameters.QValueOutputFilter).ToList();
+            FilteredPsmList.RemoveAll(b => b.IsDecoy);
+            FilteredPsmList.RemoveAll(b => b.IsContaminant);
+            //group psms by peptide and charge, the psms having same sequence and same charge will be in the same group
+            Dictionary<(String, int), List<PeptideSpectralMatch>> PsmsGroupByPeptideAndCharge = new Dictionary<(String, int), List<PeptideSpectralMatch>>();
+            foreach (var x in FilteredPsmList)
```

Comment:
Could a more clear variable be used here instead of x

Reply:
Thanks for the advice!